packages/react/src/setupTests.ts

```diff
@@ -2,6 +2,9 @@
 import '@testing-library/jest-dom';
 import 'jest-axe/extend-expect';
 import { configureAxe } from 'jest-axe';
+import { TextEncoder } from 'util';
+
+global.TextEncoder = TextEncoder;
```

Comment:
Do you know why this was needed?

Reply:
I can fix it here!

Reply:
Updated here https://github.com/dequelabs/cauldron/pull/1731/commits/9521cbc351cb3c60bd1ff1037078c0b76be7a129